.github/workflows/ios_ci.yaml

```diff
@@ -45,14 +45,27 @@ jobs:
       - uses: futureware-tech/simulator-action@v3
         id: simulator-action
         with:
-          model: 'iPhone 15'
+          model: "iPhone 15"
           shutdown_after_job: false
 
-  build-macos:
+  build-iOS:
     if: github.event.pull_request.head.repo.full_name != github.repository
     runs-on: macos-latest
 
     steps:
+      - name: Maximize build space
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+      - name: Delete GH Cache
```

Comment:
I didn't explain it clearly before. This action can only be executed locally. It would clear all the existing cache, which may cause the CI build time to be longer.